lib/diplomat/lock.rb

```diff
@@ -43,16 +46,18 @@ def wait_to_acquire key, session, value=nil, check_interval=10
     # Release a lock
     # @param key [String] the key
     # @param session [String] the session, generated from Diplomat::Session.create
+    # @param options [Hash] :dc string for dc specific query
     # @return [nil]
-    def release  key, session
+    def release  key, session, options=nil
       raw = @conn.put do |req|
         url = ["/v1/kv/#{key}"]
         url += use_named_parameter('release', session)
         url += check_acl_token
+        url += use_named_parameter('dc', options[:dc]) if options and options[:dc]
 
         req.url concat_url url
       end
-      return raw.body
+      raw.body == 'true'
```

Comment:
Yea this would be a breaking change. I made it out of consistency, but I'll save the change for the next major version bump...